history/management/commands/pull_deposits.py

```diff
@@ -35,6 +35,4 @@ def handle(self, *args, **options):
             d.status = status
             d.created_on = created_on
             d.modified_on = created_on
```

Comment:
Seems reasonable. The only downside of this approach is that we will have to remember to use `deposit_created_on` instead of `created_on`.